tests/foreman/api/test_media.py

```diff
@@ -38,10 +38,10 @@ class TestMedia:
     def class_media(self, module_org):
         return entities.Media(organization=[module_org]).create()
 
-    @pytest.mark.tier1
-    @pytest.mark.upgrade
+    @pytest.mark.tier1()
+    @pytest.mark.upgrade()
     @pytest.mark.parametrize(
-        'name, new_name',
+        ('name', 'new_name'),
```

Comment:
Does putting this in a tuple have any affect on the functionality here? I don't think so, but just curious

Reply:
this looks to have been a bug in the original test since this isn't a valid parameterization. my guess is that since we're using nailgun in the test, `name` and `new_name` being `None` result in nailgun creating values on its own.